substrate/frame/revive/src/benchmarking/mod.rs

```diff
@@ -221,7 +220,6 @@ fn default_deposit_limit<T: Config>() -> BalanceOf<T> {
 
 #[benchmarks(
 	where
-		<BalanceOf<T> as HasCompact>::Type: Clone + Eq + PartialEq + Debug + TypeInfo + Encode,
 		BalanceOf<T>: From<<pallet_balances::Pallet<T> as Currency<T::AccountId>>::Balance> + Into<U256> + TryFrom<U256>,
```

Comment:
Can we get rid of the first part here too and just keep the U256 trait bounds ?

Reply:
Yes. Good catch.